src/command_modules/azure-cli-storage/tests/test_storage_blob_scenarios.py

```diff
@@ -131,7 +126,65 @@ def get_account_key(self, group, name):
                         .format(name, group)).output
 
 
-if __name__ == '__main__':
-    import unittest
+class StorageBlobUploadLiveTests(LiveTest):
+    @live_only()
```

Comment:
Haven't find a way to let the test class itself to involve in test skipping. The `@skip` provided by `unittest` module is decorator based.